command/certificate/verify.go

```diff
@@ -65,12 +68,22 @@ Verify a certificate using a custom directory of root certificates for path vali
 '''
 $ step certificate verify ./certificate.crt --roots ./root-certificates/
 '''
+
+Verify the remaining validity of a certificate using a custom root certificate and host for path validation:
+
+'''
+$ step certificate verify ./certificate.crt --host smallstep.com --expire
```

Comment:
@mike @tashian what did we want this flag to be called? `expire`, `validity`, `verdancy`?

Reply:
`verdancy`